src/Security/test/AuthSamples.FunctionalTests/AuthSamples.FunctionalTests.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
For context -- the reason this will be important is that for source-build we will likely need to configure our repo to skip building samples b/c they have a bunch of 3rd party samples. I was planning to use `IsSampleProject` as a way to skip those projects. I want to make sure tests still build and that this doesn't get picked up in the mix unintentionally.